src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -308,14 +307,17 @@ impl AstDisplay for Connector {
                 f.write_str("'");
             }
             Connector::S3 {
-                bucket,
                 key_sources,
                 pattern,
             } => {
-                f.write_str("S3 BUCKET '");
-                f.write_str(&display::escape_single_quote_string(bucket));
-                f.write_str("' OBJECTS FROM");
-                for ks in key_sources {
+                f.write_str("S3 OBJECTS FROM");
+                let mut sources = key_sources.iter();
+                match sources.next() {
+                    Some(ks) => ks.fmt(f),
+                    None => unreachable!("Missing first key source in S3 OBJECTS"),
+                }
+                for ks in sources {
+                    f.write_str(", ");
```

Comment:
`display::comma_separated` is the convention for this around here!

Reply:
ah nice.